### DISCUSSION TITLE
SBS World Watch

### DISCUSSION THREAD

**Discussion by darrencycheung** [iptv-org/community#811] [Q&A]
### Channel ID

SBSWorldWatchSydney.au

### Notes

This channel has so many news program around the globe. This channel has news in Mandarin and Arabic.
https://th.bing.com/th/id/R.89e7af27bd7e61c21c011b99d8a28ca5?rik=aOO2cvGK93ocaA&pid=ImgRaw&r=0
It was launched on May this year. Can anyone provide an m3u8 for me please?

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by darrencycheung** [comment]
https://i.mjh.nz/au/Sydney/tv.320203000308.m3u8

**Comment by darrencycheung** [comment]
The link is geo-blocked

**Comment by Sphinxroot** [comment]
kodi only   addon playlist loader or no x-forwarded= geo


#EXTINF:-1 tvg-ID="SB1.AU" tvg-logo="https://static.epg.best/au/SBS.au.png" group-title="",SBS One
https://sbs-live.akamaized.net/hls/live/2002827/channel1/geo/master_2500K.m3u8|X-Forwarded-For=52.62.247.239
#EXTINF:-1 tvg-ID="SBV.AU" tvg-logo="https://static.epg.best/au/SBSViceland.au.png" group-title="",SBS Viceland
https://sbs-live.akamaized.net/hls/live/2002828/channel2/geo/master_2500K.m3u8|X-Forwarded-For=52.62.247.239
#EXTINF:-1 tvg-ID="SBF.AU" tvg-logo="https://d2po7v53a8yrck.cloudfront.net/austatic/media/ad6c5b49-3c34-4a6d-ab5b-d1bd2eea8fda.png" group-title="",SBS Food
https://sbs-live.akamaized.net/hls/live/2002829-b/channel3/geo/master_1800K.m3u8|X-Forwarded-For=52.62.247.239
#EXTINF:-1 tvg-ID="NIT.AU" tvg-logo="https://static.epg.best/au/NITV.au.png" group-title="Australia",NITV
https://sbs-live.akamaized.net/hls/live/2002826/channel4/geo/master_1800K.m3u8|X-Forwarded-For=52.62.247.239
#EXTINF:-1 tvg-ID="SBM.AU" tvg-logo="https://d2po7v53a8yrck.cloudfront.net/austatic/media/3374ed8c-8b84-4703-8449-7142e3bd838e.png" group-title="",SBS World Movies
https://sbs-live.akamaized.net/hls/live/2005470/channel5/geo/master_2500K.m3u8|X-Forwarded-For=52.62.247.239

**Comment by freearhey** [comment]
https://i.mjh.nz/au/Sydney/tv.320203000308.m3u8 (offline)
<img width="364" alt="image" src="https://user-images.githubusercontent.com/7253922/232234686-6460bbb8-2c35-47fc-8344-a46de1ef1eca.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19781
